packages/nextjs/hooks/scaffold-stark/useSwitchNetwork.ts

```diff
@@ -1,15 +1,24 @@
+const getChainId = (network: string): string => {
+  switch (network) {
+    case "mainnet":
+      return "SN_MAIN";
+    case "devnet":
+      return "SN_GOERLI";
+    default:
+      return `SN_${network.toUpperCase()}`;
+  }
+};
+
 export const useSwitchNetwork = () => {
```

Comment:
Argent works well but facing an issue with Bravoos

https://www.loom.com/share/397e134da9434673bb8811cfa4bbd2b3?sid=c97ca01b-cc52-4731-b0ac-a266c87a737a

Reply:
Can u check information of "window.starknet" and you can see the reason. This is what I was trying to figure out, is there any other way that is better for both wallets

Reply:
no `request()` function for bravoos?

Reply:
![image](https://github.com/user-attachments/assets/2e829ab7-19ec-444f-be5c-b43d8bb13c20)

Reply:
cc @jrcarlos2000

Reply:
this was the same issue we faced at the beginning on first iteration, can someone get in touch with Braavos team for this ? @Buuh2511 @metalboyrick

Reply:
i have never contact to Braavos team before. but i will try, can i ask for instructions on how to do it

Reply:
discord : https://discord.com/invite/9Ks7V5DN9z

Reply:
hey ser, any updates on this? @Buuh2511

Reply:
the braavos team said this is on the list but not done yet. we should keep this open for now see how it goes 👍